apps/src/templates/HomeBanner.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from './Button';
+import color from '../util/color';
+
+export default class HomeBanner extends React.Component {
```

Comment:
The naming is a bit confusing to me in that there are two components that are similarly-named but it's hard to tell how they are different from the name.  Do we have some naming convention for top-level components vs. non top-level components?  If not, maybe call this something like "CovidBanner" or "Special2020Banner"?

Reply:
good call, I struggled with the naming. I named it "HomeBanner" because it has a home icon, but I think Special2020Banner will help differentiate things better.